File: mobile/projeto_gerenciamento_pedido/lib/model/Funcionario.dart
class Funcionario {
  int id;
  String nome;
  String sobrenome;
  String telefone;
  String cpf;
  String login;
  String senha;
  bool isPermissao;
  String tipoAcesso;

  bool isReset;
  bool isLogado;

  Funcionario(
      {this.id,
      this.nome,
      this.sobrenome,
      this.telefone,
      this.cpf,
      this.login,
      this.senha,
      this.isPermissao,
      this.tipoAcesso,
      this.isReset,
      this.isLogado});

  Funcionario.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    nome = json['nome'];
    sobrenome = json['sobrenome'];
    telefone = json['telefone'];
    cpf = json['cpf'];
    login = json['login'];
    senha = json['senha'];
    isPermissao = json['isPermissao'];
    tipoAcesso = json['tipoAcesso'];

    isReset = json['isReset'];
    isLogado = json['isLogado'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['nome'] = this.nome;
    data['sobrenome'] = this.sobrenome;
    data['telefone'] = this.telefone;
    data['cpf'] = this.cpf;
    data['login'] = this.login;
    data['senha'] = this.senha;
    data['isPermissao'] = this.isPermissao;
    data['tipoAcesso'] = this.tipoAcesso;

    data['isReset'] = this.isReset;
    data['isLogado'] = this.isLogado;
    return data;
  }
}

File: software_desktop/SistemaGerenciamentoPedidos/src/model/Funcionario.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import java.util.Date;

/**
 *
 * @author Flavio
 */
public class Funcionario {
    private int id;
    private String nome;
    private String sobrenome;
    private String telefone;
    private String cpf;
    private String login;
    private String senha;
    private boolean isPermissao;
    private String tipoAcesso;
    private Date ultimoAcesso;
    private boolean isLogado;
    private boolean isReset;
    
    public Funcionario(){
}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getSobrenome() {
        return sobrenome;
    }

    public void setSobrenome(String sobrenome) {
        this.sobrenome = sobrenome;
    }

    public String getTelefone() {
        return telefone;
    }

    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }

    public String getCpf() {
        return cpf;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public String getSenha() {
        return senha;
    }

    public void setSenha(String senha) {
        this.senha = senha;
    }

    public boolean isIsPermissao() {
        return isPermissao;
    }

    public void setIsPermissao(boolean isPermissao) {
        this.isPermissao = isPermissao;
    }

    public String getTipoAcesso() {
        return tipoAcesso;
    }

    public void setTipoAcesso(String tipoAcesso) {
        this.tipoAcesso = tipoAcesso;
    }

    public Date getUltimoAcesso() {
        return ultimoAcesso;
    }

    public void setUltimoAcesso(Date ultimoAcesso) {
        this.ultimoAcesso = ultimoAcesso;
    }

    public boolean isIsLogado() {
        return isLogado;
    }

    public void setIsLogado(boolean isLogado) {
        this.isLogado = isLogado;
    }

    public boolean isIsReset() {
        return isReset;
    }

    public void setIsReset(boolean isReset) {
        this.isReset = isReset;
    }
    
    
    
}
